AspNetCore.Demo/Startup.cs

```diff
@@ -25,10 +31,28 @@ public class Startup {
                 "/mirrorsharp",
                 new MirrorSharpOptions {
                     SelfDebugEnabled = true,
-                    IncludeExceptionDetails = true
+                    IncludeExceptionDetails = true,

```

Comment:
I normally avoid dangling commas, not sure if there is a an editorconfig setting for it...